CMakeLists.txt

```diff
@@ -98,7 +121,17 @@ if(NOT EXISTS "${CMAKE_HOME_DIRECTORY}/3rdparty/edbee-lib/CMakeLists.txt")
   endif()
 endif()
 
-if(WITH_UPDATER)
+# We do not use CMake build process on AppVeyor but if we did we would need to
+# include a test and install of the lua-code-formatter "lcf" module because we
+# would need to have it in the binary created so that it would be available.
+# Normal users should install "luarocks" (the lua package/module manager) and
+# get the module with a OS specific command - on linux it could be
+# "sudo luarocks install lcf"
```

Comment:
That is not the case, unfortunately luarocks will install you the 5.3 version while we need 5.1 - and we also require modifications to the formatter, so the luarocks comment is misleading :thinking:

Reply:
Yeah, it would seem you need to change to `./3rdparty/lua_code_formatter` and do a `sudo luarocks make ./lcf-scm-1.rockspec` to make the module from the files in the repository, and install it into the system luarock tree (this will I think also work, without `sudo`, similarly on Windows)...